runtime/Cpp/runtime/src/RuleContext.cpp

```diff
@@ -123,21 +104,15 @@ std::string RuleContext::toString(const std::vector<std::string> &ruleNames, Rul
   return ss.str();
 }
 
-std::string RuleContext::toString() {
-  return toString(nullptr);
-}
+std::string RuleContext::toString() { return toString(nullptr); }
 
-std::string RuleContext::toString(Recognizer *recog) {
-  return toString(recog, &ParserRuleContext::EMPTY);
-}
+std::string RuleContext::toString(Recognizer *recog) { return toString(recog, &ParserRuleContext::EMPTY); }
 
 std::string RuleContext::toString(Recognizer *recog, RuleContext *stop) {
   if (recog == nullptr)
-    return toString(std::vector<std::string>(), stop); // Don't use an initializer {} here or we end up calling ourselve recursivly.
+    return toString(std::vector<std::string>(),
+                    stop); // Don't use an initializer {} here or we end up calling ourselve recursivly.
```

Comment:
Same situation: the trailing comment makes the line too long. In such cases better move it to before the line it applies to, instead of wrapping that line. But of course this is something that needs manual editing. Has nothing to do with formatting rules.